ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/verify.yml

```diff
@@ -24,3 +24,24 @@
       delegate_to: 127.0.0.1
       ansible.builtin.import_role:
         name: arista.avd.eos_cli_config_gen
+
+- name: Converge
```

Comment:
Please don't modify the platbooks here for tests. It is better to have two hosts under eos_designs_unit_tests testing each behavior specifically, and with minimal other config.